mlrun/model.py

```diff
@@ -314,6 +423,12 @@ def credentials(self) -> Credentials:
     def credentials(self, credentials):
         self._credentials = self._verify_dict(credentials, "credentials", Credentials)
 
+    def to_dict(self, fields: list = None, exclude: list = None, strip: bool = False):
```

Comment:
not sure why this is needed. you add the `self._fields_to_strip` to the excluded list in the base class anyway.

Reply:
redundant, removed